src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,17 @@
-@mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
-  &:hover {
-    #{$_property}: $_toValue;
+@mixin tablet {
+  @media screen and (min-width: 768px) {
+    @content;
+  }
+}
+
+@mixin desktop {
```

Comment:
The 'desktop' mixin uses a min-width of 1280px. This is not always the correct breakpoint for desktops. It's better to use a variable for this and adjust as needed per project basis.